### ISSUE TITLE
VisibleDeprecationWarning: Passing `normed=True` on non-uniform bins has always been broken

### ISSUE THREAD

**Issue by scls19fr** [python-windrose/windrose#108]
https://travis-ci.org/python-windrose/windrose/jobs/447013574#L847

```
VisibleDeprecationWarning: Passing `normed=True` on non-uniform bins has always been broken, and computes neither the probability density function nor the probability mass function. The result is only correct if the bins are uniform, when density=True will produce the same result anyway. The argument will be removed in a future version of numpy.
  hist, bins = np.histogram(var, bins=bins, normed=True)
```

**Comment by HonestyBrave** [comment]
https://www.pythonf.cn/read/85868

you could replace "normed" with "density"

**Comment by scls19fr** [comment]
PR much appreciated!

**Comment by scls19fr** [comment]
Closed by #176
